sdks/python/apache_beam/transforms/display.py

```diff
@@ -86,13 +86,17 @@ def _populate_items(self, display_data_dict):
     """ Populates the list of display data items.
     """
     for key, element in display_data_dict.items():
+      if element is None:
```

Comment:
In the Java SDK, we take a strict stance against registering null display data. Implicitly dropping null values in the SDK can lead to confusing behavior (_"I registered this display item, why isn't it showing up?"_).  Instead, the Java SDK has a helper method `.addIfNotNull(..)`, and the default `.add(..)` method throws if a null value is provided. This makes it very obvious whether you can reliably expect your display item to make it to the UI and when it might be dropped.

I recommend following this semantic here in the spirit of "element of least-surprise", but also so we are consist across SDKs.